src/solver/conv_ocl_dir2Dfwd_exhaustive_search.cpp

```diff
@@ -253,14 +255,14 @@ ConvOclDirectFwdLegacyExhaustiveSearch::SearchImpl(const ExecutionContext& ctx,
 
 #if MIOPEN_ALLOC_BUFFERS
```

Comment:
This `MIOPEN_ALLOC_BUFFERS` is a leftover of functionality removed some time ago. I highly recommend removing all this stuff. This way, the `rand()` calls can simply be removed.

Reply:
@JehandadKhan @CAHEK7 ☝️